packages/provider-elasticsearch/src/example-types/metricGroups/pivot.js

```diff
@@ -289,23 +289,38 @@ let createPivotScope = (node, schema, getStats) => {
           // The API of `{sort: {field, direction}}` is respected by fieldValues and can be added to others
           group.sort = { field: sortField, direction: sort.direction }
         }
-        let build = lookupTypeProp(_.identity, 'buildGroupQuery', group.type)
-        //Remove anything that needs to be hoisted before composing further
-        // We are iterating through the groups reversed so we need to subtract instead of add to get the right index
-        let reversedLookupIndex = groups.length - index - 1
+        let build = lookupTypeProp(
+          _.rearg([1], _.identity),
```

Comment:
I'm not sure we should do this at all. This change fixes a bug to return `children` instead of incorrectly returning `group` when pivot groupings don't specify `buildGroupQuery`, but when would that be desirable? Should the default case throw an error?

Reply:
Agreed, replacing it with a throw function instead